pandas/tests/frame/test_constructors.py

```diff
@@ -717,6 +717,18 @@ def test_constructor_period_dict(self):
         assert df["a"].dtype == a.dtype
         assert df["b"].dtype == b.dtype
 
+    @pytest.mark.parametrize(
+        "data,dtype",
+        [
+            (pd.Period("2012-01", freq="M"), "period[M]"),
+            (pd.Period("2012-02-01", freq="D"), "period[D]"),
```

Comment:
do we have sufficient coverage in a Series for this?

Reply:
Not sure what you are asking here?

Reply:
I mean if we do ```pd.Series({0: pd.Period(...)})``` is this tested?

Reply:
It looks like that already works correctly on the existing master branch and is unaffected by the changes of this PR. 

```python
pd.Series({0: pd.Period('2020-01')})

0    2020-01
dtype: period[M]
```

But I don't know where the Series construction is tested, so I'm not sure if there is coverage yet. I'm happy to do whatever but would need more info from you on where to check for existing tests / where to create a new one.

Reply:
Series tests are in `pandas/tests/series/test_period.py` and also 
https://github.com/pandas-dev/pandas/blob/9820c426a035495ca14dfb0fefe873e96f7483e0/pandas/tests/series/test_constructors.py#L1022-L1060
I didn't see `pd.Series({0: pd.Period(...)})` but might have missed it

Reply:
Great, thank you, I added a test